smart-contracts/reports/gas.js

```diff
@@ -19,6 +20,7 @@ contract('Reports', accounts => {
   })
 
   it('gas usage report', async () => {
+    expect.assertions(1)
```

Comment:
this is causing the build fail.  not sure where this line came from, but not valid syntax for these tests

Reply:
Yeah, this is `jest` and a pretty useful thing that we enforce on all other tests. I would be on favor of moving over to jest instead of mocha too for smart contract but not sure what is involved.